index.bs

```diff
@@ -2512,38 +2531,38 @@ navigation status</dfn> struct, which has the following items:
 
 <pre class="cddl remote-cddl">
 BrowsingContextCommand = (
-  BrowsingContextCaptureScreenshotCommand //
-  BrowsingContextCloseCommand //
-  BrowsingContextCreateCommand //
-  BrowsingContextGetTreeCommand //
-  BrowsingContextHandleUserPromptCommand //
-  BrowsingContextNavigateCommand //
-  BrowsingContextReloadCommand
+  browsingContext.CaptureScreenshot //
+  browsingContext.Close //
+  browsingContext.Create //
+  browsingContext.GetTree //
+  browsingContext.HandleUserPrompt //
+  browsingContext.Navigate //
+  browsingContext.Reload
 )
 </pre>
 
 [=local end definition=]
 
 <pre class="cddl local-cddl">
 BrowsingContextResult = (
-  BrowsingContextCaptureScreenshotResult //
-  BrowsingContextCreateResult //
-  BrowsingContextGetTreeResult //
-  BrowsingContextNavigateResult
+  browsingContext.CaptureScreenshotResult //
+  browsingContext.CreateResult //
+  browsingContext.GetTreeResult //
+  browsingContext.NavigateResult
 )
 
 BrowsingContextEvent = (
-  BrowsingContextCreatedEvent //
-  BrowsingContextDestroyedEvent //
-  BrowsingContextNavigationStartedEvent //
-  BrowsingContextFragmentNavigatedEvent //
-  BrowsingContextDomContentLoadedEvent //
-  BrowsingContextLoadEvent //
-  BrowsingContextDownloadWillBeginEvent //
-  BrowsingContextNavigationAbortedEvent //
-  BrowsingContextNavigationFailedEvent //
-  BrowsingContextUserPromptClosedEvent //
-  BrowsingContextUserPromptOpenedEvent
+  browsingContext.ContextCreated //
+  browsingContext.ContextDestroyed //
+  browsingContext.NavigationStarted //
+  browsingContext.FragmentNavigated //
+  browsingContext.DomContentLoaded //
+  browsingContext.Load //
```

Comment:
Agree with @whimboo that having `Event` suffix would be nice, but even in this version it became more readable.

Reply:
I actually wonder how it would look like for a possible code generating tool. For the classes as generated it probably would make sense to have the `Command` or `Event` suffix included. If we don't have it here maybe some hand-crafted code is needed to detect that via the parent type which would be `BrowsingContextEvent` here.

Reply:
Well any code generating tool is going to have to introduce its own logic since the mapping from CDDL to types isn't trivial.

However as a point of reference, geckodriver doesn't suffix command names with `Command`: https://searchfox.org/mozilla-central/source/testing/geckodriver/marionette/src/webdriver.rs#151, and I don't think also having events would change that.

Reply:
Right, but there was also no need to differentiate between commands and events given that the latter don't exist.

Maybe we can put it on the agenda for next week to finalize the discussion?

Reply:
Well, as I said, if I were writing this in a similar style in Rust I wouldn't change that; a commands would match a variant of the `Command` enum, events would match a variant of the `Event` enum, and the actual data would be in the `*Paramaters` types.

Maybe in some other language you'd uae a different pattern, but from the point of view of the spec we should consider what makes the spec easy to read and write, not what implementation languages should use. From my point of view, the `module.MethodName` types are already quite verbose and so `module.MethodNameCommand` would be even worse.

For comparison, the CDP protocol definitions don't suffix commands/events/types names with anything (e.g. you just get `Page.navigate` and `Page.loadEventFired` rather than `Page.navigateCommand` and `Page.loadEventFiredEvent`).

Reply:
Those are actually good arguments @jgraham and you convinced me! I'm happy with the proposed changes, and will resolve this issue.